src/mlx/warnings/warnings_checker.py

```diff
@@ -32,41 +32,32 @@ def substitute_envvar(checker_config, keys):
 
 class WarningsChecker:
     name = 'checker'
+    logging_fmt = "{checker_name}: {message}"
 
-    def __init__(self, verbose=False):
+    def __init__(self, verbose, output):
         ''' Constructor
 
         Args:
             name (str): Name of the checker
             verbose (bool): Enable/disable verbose logging
         '''
         self.verbose = verbose
+        self.output = output
```

Comment:
I've never liked the args `verbose, output` here. Can they become args of `init_loggers` instead now?

Reply:
Resolved in 65052edb2be6ab8576d9fdfe1c7aa3905cacd2db